pkg/authorization/cache/indexer_interface_implementations.go

```diff
@@ -0,0 +1,148 @@
+package cache
+
+import (
+	apierrors "github.com/GoogleCloudPlatform/kubernetes/pkg/api/errors"
+	utilerrors "github.com/GoogleCloudPlatform/kubernetes/pkg/util/errors"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+)
+
+type policies struct {
+	policyCache *IndexerPolicyCache
+}
+
+func (this *policies) List() (*authorizationapi.PolicyList, error) {
+	policyList := &authorizationapi.PolicyList{}
+	returnedList := this.policyCache.policyIndexer.List()
+	for i := range returnedList {
+		policy, castOK := returnedList[i].(*authorizationapi.Policy)
+		if !castOK {
+			return policyList, apierrors.NewInvalid("PolicyList", "policyList", []error{})
+		}
+		policyList.Items = append(policyList.Items, *policy)
+	}
+	return policyList, nil
+}
+
+func (this *policies) Get(name string) (*authorizationapi.Policy, error) {
+	errors := []error{}
+	item, exists, getErr := this.policyCache.policyIndexer.Get(name)
```

Comment:
This reads weird to me, but I haven't dug into the code yet.  I'd expect the indexer to be unable to distinguish conflicting names across namespaces.  @derekwaynecarr ?